### PR TITLE
Implement computeHorizontalScrollRange related API

### PR DESCRIPTION
**Pull request by shaoboyan**
Test case of these APIs will be covered by a sample in next PR.
BUG=XWALK-4394

### CODE REVIEW

**Comment by shaoboyan** [comment]
@XiaosongWei @axinging   PTAL.

**Comment by axinging** [comment]
Two comments:
1), XWalkContentsClient is not public to customer. So how customer use mContentViewClient.onOverScrolled referred https://github.com/crosswalk-project/chromium-crosswalk/pull/330?

2), Please provide test case for this.

**Comment by shaoboyan** [comment]
@axinging First, onOverScrolled is not used directly used like mContentViewClient.onOverScrolled, the following patch which enable onOverScroll on crosswalk side will describe it. But if the PR in chromium-crosswalk don't merge. The following PR will cause build failure.
Secondly, I've developed a sample which cover overscroll related APIs and compute related APIs and it will be posted as a stand-alone PR.

File: runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java [hunk 1]
```diff
@@ -1447,6 +1447,54 @@ public void scrollBy(int x, int y) {
     }
 
     /**
+     * Compute the horizontal range that the horizontal scrollbar represents.
+     * @return the range of horizontal scrollbar.
+     */
```
Comment (axinging) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java]: Please API version, like @since  x.0. And below too.

**Comment by axinging** [comment]
Thanks. I mean test case like these: src/xwalk/test/android/core/javatests/src/org/xwalk/core/xwview/test/

**Comment by shaoboyan** [comment]
@axinging  since it's not easy to add auto-test case for these API, I prefer to cover these api and overscroll API all in a sample case. thx.

File: runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java (hunk 1 above)
Comment (shaoboyan) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java]: done! thx.

**Comment by axinging** [comment]
LGTM

**Comment by XiaosongWei** [comment]
Please add [Android] tag to the begin of commit/PR, others LGTM.

**Comment by shaoboyan** [comment]
@XiaosongWei Added [Android] Tag on commit log. thx.

**Comment by shaoboyan** [comment]
@lincsoon Pls help merge this PR. thx.

**Pull request merged**